weather/ngsild-payloads/weatherAlert.jsonld

```diff
@@ -0,0 +1,58 @@
+{
+	"id": "urn:ngsi-ld:WeatherAlert:Menton",
+	"type": "WeatherAlert",
+	"weatherRisk": [
+		{
+			"type": "Property",
+			"value": "Des averses d'intensités modérées",
```

Comment:
should the value be this free text or the risk name (or risk code)?

Reply:
I would see the "subdivision texts" into a sub-attribute. There even seem to have some sort of structure (Qualification / Observations notables / Évolutions prévues)